src/lib/types.ts

```diff
@@ -150,6 +150,34 @@ export const postgresPublicationSchema = Type.Object({
 })
 export type PostgresPublication = Static<typeof postgresPublicationSchema>
 
+export const postgresTriggerSchema = Type.Object({
+  id: Type.Integer(),
+  enabled_mode: Type.Union([
+    Type.Literal('ORIGIN'),
+    Type.Literal('REPLICA'),
+    Type.Literal('ALWAYS'),
+    Type.Literal('DISABLED'),
+  ]),
+  name: Type.String(),
+  table: Type.String(),
+  schema: Type.String(),
+  condition: Type.String(),
```

Comment:
Sorry I wasn't very clear, in the prev. review I meant that `condition?:` should be `condition:`, but it should still be nullable to indicate that there's no condition.

Reply:
@soedirgo no i understood what you meant i just thought that it was better that condition always return a string instead. let me go and update